contrib/freshmaker-cli

```diff
@@ -224,6 +229,45 @@ def get_freshmaker_events_by_ids(api_url, ids):
     events = sorted(events, key=lambda k: k['id'])
     return events
 
+def get_advisories_from_errata():
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED state. Returns the
+    list of advisories that are IMPORTANT or CRITICAL.
+
+    """
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get("https://errata.devel.redhat.com/filter/2943.json", verify=False, auth=krb_auth)
```

Comment:
The URL should go to configuration.

Reply:
@hluk , what do mean? 
We have no API to list advisories with filters. So I've created the filter.

Reply:
I thought that there is a configuration, e.g. for different environments (env/stage/prod). The URL should be at least a constant at the top of the script.

Reply:
Added